### DISCUSSION TITLE
Install multiple programs at once

### DISCUSSION THREAD

**Discussion by rtomik** [PSAppDeployToolkit/PSAppDeployToolkit#706] [Q&A] [answered]
Hi,
for installing like 10 programs for new deployment should i create main script for every program or can add it everything into one script? 
Thanks

**Comment by cyborgrd** [comment]
you can put all in one deployment script

**Comment by That-Annoying-Guy** [comment]
That depends on you.

It's a script so you can do pretty much what you want and stuff in between

But I would NOT do one script for like Office+Teams+Adobe Reader+AntiVirus+Browser.
You'll want to manage apps like that separately. In SCCM (now called MECM ), 
we use a task sequence to install many packages of whatever type in a row.

FYI: This is not the main discussion area. go to https://discourse.psappdeploytoolkit.com/c/thetoolkit/general-discussion

**Comment by rtomik** [comment]
okay thanks i will try it out. Are there more examples with this script?

**Comment by That-Annoying-Guy** [comment]
look in https://discourse.psappdeploytoolkit.com/c/thetoolkit/deployment-scripts

**Comment by mjr4077au** [comment, marked as answer]
PSADT v4 is going to support advanced situations like concurrent sessions, where you install app after app each with their own logging. You can also start a new session mid-session for determined dependencies, etc. I must be clear that this doesn't not mean parallel installations, it's still single threaded, but it addresses the discussion here
